src/modules/DecodeQr/info.json

```diff
@@ -5,8 +5,8 @@
   },
   "outputs": {
     "qrval": {
-      "type": "string"
+      "type": "button"
```

Comment:
The output type should still be a string.

If the string is an output field instead of an input field, just append a `span` or `p` or `b` instead of `input` HTML element. And next to the output field, there should be a copy button (preferably with a nice looking icon).

Reply:
I appended a button to the div element showing the output field hope that is fine

Reply:
![Screenshot from 2020-01-15 20-00-23](https://user-images.githubusercontent.com/32903329/72442008-b99d2e80-37d1-11ea-833f-0237824b130e.png)